tests/Facility.ConformanceApi.UnitTests/CodeGenTests.cs

```diff
@@ -25,5 +26,12 @@ public void GenerateConformanceApi()
 			var output = generator.GenerateOutput(service);
 			output.Files.Count(x => x.Name == "IConformanceApi.g.cs").Should().Be(1);
 		}
+
+		[Test]
+		public void AvoidsCrashOnDefaultEnumComparison()
```

Comment:
The relevant test is that `ToString` doesn't crash on a default value.

Comparison currently happens to call `ToString`, but the calls to `ToString` in `Equals` and `GetHashCode` should really be changed to `m_value`, since getting the cached value isn't necessary there.

Reply:
As an aside, perhaps we should introduce `AnswerTests` or `EnumTests`. It would be nice to also demonstrate and test the `switch` strategy that is enabled by `Strings`.